pkg/types/target.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	// LicenseScanner is the scanner of licenses
 	LicenseScanner = Scanner("license")
+
+	// SbomScanner includes all found packages on report
+	SbomScanner = Scanner("sbom")
```

Comment:
```suggestion
	SBOMScanner = Scanner("sbom")
```

Reply:
Changed in https://github.com/aquasecurity/trivy/pull/5255/commits/6b6a6cc9dabd301a85d912eb4d542081ee2ce8eb